### PR TITLE
tests: removed resty-cli from the test depedencies.

### PR DESCRIPTION
**Pull request by spacewander**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-resty-core project.

### CODE REVIEW

File: t/pipe.t [hunk 1]
```diff
@@ -1358,11 +1358,51 @@ lua pipe kill process:
...
+                                os.exit(0)
+                            end)
+                        }
+                    }
+                ]]
+
+                assert(os.execute("mkdir -p $TEST_NGINX_HTML_DIR/logs"))
+
+                local conf_file = "$TEST_NGINX_HTML_DIR/nginx.conf"
+                local f, err = io.open(conf_file, "w")
```
Comment (agentzh) [inline t/pipe.t]: Better use Test::Nginx's `--- user_files` feature to prepare such files? Is that possible?

**Comment by agentzh** [comment]
@spacewander travis ci is failing?

**Comment by thibaultcha** [comment]
Could we apply the following patch in order to stay consistent in our setting of the `lua_package_path` directive?

```diff
diff --git a/t/pipe.t b/t/pipe.t
index 17e3298..aebf346 100644
--- a/t/pipe.t
+++ b/t/pipe.t
@@ -32,7 +32,7 @@ _EOC_
 });
 
 $ENV{TEST_NGINX_HTML_DIR} ||= html_dir();
-$ENV{TEST_NGINX_LIB_DIR} ||= $t::TestCore::pwd;
+$ENV{TEST_NGINX_LUA_PACKAGE_PATH} = "$t::TestCore::lua_package_path";
 
 env_to_nginx("PATH");
 no_long_string();
@@ -1394,7 +1394,7 @@ daemon off;
 master_process off;
 worker_processes 1;
 http {
-    lua_package_path "$TEST_NGINX_LIB_DIR/lib/?.lua;;";
+    lua_package_path "$TEST_NGINX_LUA_PACKAGE_PATH";
     init_worker_by_lua_block {
         ngx.timer.at(0, function()
             require "ngx.pipe".spawn{"no-such-cmd"}:wait()
```

**Comment by thibaultcha** [comment]
Applied the patch and committed to master, thanks. Glad we got rid of this one.

**Pull request closed without merging**